charts/argo-templates/templates/diff-bq-table.yaml

```diff
@@ -22,6 +22,7 @@ spec:
           - name: jade-bq-dataset
           - name: upsert
           - name: diff-full-history
+          - name: jade-table-pre-filter
```

Comment:
For most cases, this parameter will have the value of empty string.
For ClinVar, it will be `release_date = '<some-value>'`